### DISCUSSION TITLE
How to access build/deployment logs in UI?

### DISCUSSION THREAD

**Discussion by nyxb** [Dokploy/dokploy#755] [Q&A] [answered]
I might be missing something obvious, but I'm having trouble finding the build and deployment logs in the Dokploy UI. If this functionality exists, I would greatly appreciate guidance on where to find it.

Current situation:
When deploying a project using compose, I can only observe:
- A notification in the bottom right that deployment started
- Eventually, a red dot indicating an error if deployment fails
- The logs view asks me to select a running container

![NyxbShot-2024-11-24-at-11 35 27@2x](https://github.com/user-attachments/assets/6ec4a24a-767c-4c5d-9f74-349086c639c0)
![NyxbShot-2024-11-24-at-11 36 21@2x](https://github.com/user-attachments/assets/ef6d03aa-3e28-4164-b7dc-88021e3ce555)
![NyxbShot-2024-11-24-at-11 36 37@2x](https://github.com/user-attachments/assets/5c001123-f936-4860-9aa5-7d6fa5af66d0)

What I'm trying to find (and might be missing):
- Live build/deployment process output
- Terminal logs during the deployment
- Error messages when deployment fails

For comparison: In Coolify, I can see the live terminal output during deployment, which helps tremendously with debugging. I'm wondering if Dokploy has similar functionality that I'm just not finding.

Specific questions:
1. Is there a way to view the live build/deployment process in the UI?
2. If deployment fails and no containers are running, how can I access the logs to understand what went wrong?
3. If these features exist, could someone point me to where I can find them in the UI?

If this functionality doesn't exist yet, would it be possible to consider adding it? It would be very helpful for debugging deployment issues.

Thank you in advance for any guidance or clarification!

**Comment by Siumauricio** [comment, marked as answer]
As far as I see you are trying to deploy an application on an external server, if you have misconfigured your server or do not have access will not create any log, why? because if you do not have access to the remote server can not create any log or write it on the remote server, what I suggest is that you check well your servers that you can connect through the configuration of servers section

1. Yes
2. This is basically in the deployments tab there you will see all the deployments whether it has errors/running or done.
3. This is already implemented since dokploy version 0.0.1.

I strongly recommend you to read this if you are using multi server https://docs.dokploy.com/docs/core/multi-server/instructions

If you are not using multi server, it would be ideal to see the server logs in settings -> server -> show logs

**Reply by nyxb** [reply]
Ah yes, that was exactly it! It was indeed because I had set up multi-server and didn't notice that not all steps were completed during the server setup. After resolving some permission issues, everything is working perfectly now.
Thank you so much for your helpful support! 👍
